config/application.rb

```diff
@@ -6,19 +6,25 @@
 # you've limited to :test, :development, or :production.
 Bundler.require(*Rails.groups)
 
-class Application < Rails::Application
-  # Settings in config/environments/* take precedence over those specified here.
-  # Application configuration should go into files in config/initializers
-  # -- all .rb files in that directory are automatically loaded.
+module Application
```

Comment:
Forgot to say that the new configuration generated by rails 4.2 put the Application class inside a module. That's why I got this complain.